modules/environment-roles/templates/shared_terraform_policy_2.json.tpl

```diff
@@ -180,6 +108,18 @@
         "s3:PutObjectVersionTagging"
       ],
       "Resource": "*"
+    },
+    {
+      "Sid": "apigateway",
+      "Effect": "Allow",
+      "Action": [
+        "apigateway:POST",
```

Comment:
Remind me - are these shared Terraform policies the ones used by Jenkins deployments? If so, do they need to be able to call API Gateway directly? I'd have thought they'd just need permission to deploy API Gateway changes. Sorry if I've misunderstood how these policies work.

Reply:
In a slightly confusing move by AWS, these are the policies to deploy the changes https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-control-access-using-iam-policies-to-create-and-manage-api.html

Reply:
I'd normally suggest adding a comment but I'm guessing you can't in a JSON template, since JSON doesn't support comments?

Can we made the SID "ApiGatewayDeployment" or something in the hope of making it a bit clearer?

Reply:
Yeah, makes sense, I've changed it.